apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -75,64 +86,60 @@ const AccessManagement = (): JSX.Element => {
 		onOpen: onDeleteModalOpen,
 	} = useModal();
 
-	const [selectedGroupId, setSelectedGroupId] = useState<null | number>(null);
-
-	const hasSelectedGroup = Boolean(selectedGroupId);
-
-	useEffect(() => {
+	const fetchUsers = useCallback(() => {
 		void dispatch(
 			userActions.loadAll({ page: userPage, pageSize: userPageSize }),
 		);
-	}, [dispatch, userPage, userPageSize, onGroupPageChange]);
+	}, [dispatch, userPage, userPageSize]);
 
-	useEffect(() => {
+	const fetchGroups = useCallback(() => {
 		void dispatch(
 			groupActions.loadAll({ page: groupPage, pageSize: groupPageSize }),
 		);
-	}, [dispatch, groupPage, groupPageSize, onGroupPageChange]);
+	}, [dispatch, groupPage, groupPageSize]);
 
 	useEffect(() => {
-		if (groupDeleteStatus === DataStatus.FULFILLED) {
-			const maxGroupPages = Math.ceil(groupsTotalCount / groupPageSize);
-			const maxUserPages = Math.ceil(usersTotalCount / userPageSize);
-			const validGroupPage = Math.min(groupPage, maxGroupPages);
-			const validUserPage = Math.min(userPage, maxUserPages);
-			onGroupPageChange(validGroupPage);
-			onUserPageChange(validUserPage);
-		}
-	}, [
-		groupDeleteStatus,
-		groupPage,
-		groupsTotalCount,
-		groupPageSize,
-		onGroupPageChange,
-		userPage,
-		usersTotalCount,
-		userPageSize,
-		onUserPageChange,
-	]);
+		fetchUsers();
+	}, [fetchUsers]);
+
+	useEffect(() => {
+		fetchGroups();
+	}, [fetchGroups]);
 
 	const handleGroupCreateSubmit = useCallback(
 		(payload: GroupCreateRequestDto): void => {
-			void dispatch(
-				groupActions.create({
-					payload,
-					query: { page: userPage, pageSize: userPageSize },
-				}),
-			);
-			onModalClose();
+			void dispatch(groupActions.create(payload));
 		},
-		[dispatch, onModalClose, userPage, userPageSize],
+		[dispatch],
 	);
 
+	const [groupToEdit, setGroupToEdit] =
+		useState<GroupGetAllItemResponseDto | null>(null);
+	const hasGroupToEdit = groupToEdit !== null;
+
+	const handleGroupUpdateSubmit = useCallback(
+		(id: number, payload: GroupUpdateRequestDto) => {
+			void dispatch(groupActions.update({ id, payload }));
+		},
+		[dispatch],
+	);
+
+	const onEdit = useCallback(
```

Comment:
handlers created inside component should be named with handle
```suggestion
	const handleEdit = useCallback(
```